plugins/tpm/fu-tpm-v2-device.c

```diff
@@ -343,6 +343,10 @@ fu_tpm_v2_device_setup(FuDevice *device, GError **error)
 	tmp = fu_tpm_v2_device_convert_manufacturer(manufacturer);
 	fu_device_set_vendor(device, tmp != NULL ? tmp : manufacturer);
 
+	/* Pluton */
+	if (g_strcmp0(manufacturer, "MSFT") == 0 && g_strcmp0(model, "Pluton.TPM.A") == 0)
+		fu_device_add_internal_flag(device, FU_DEVICE_INTERNAL_FLAG_HOST_CPU_CHILD);
```

Comment:
could we do this in a quick file?

Reply:
How?

Reply:
I guess maybe matching a guid, I'll look and see if one makes sense.

Reply:
Can do something like:

    [TPM\VEN_MSFT&MOD_Pluton.TPM.A&VER_2.0]
    Flags = host-cpu-child

perhaps?